Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -193,8 +193,14 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
+
+  std::unique_ptr<IFileInfo> file_info = file_system->FindFileInfo("opening.bnr");
+  if (!file_info)
+    return;
 
+  size_t file_size = (size_t)file_info->GetSize();
```

Comment:
`size_t` is guaranteed to be able to safely store the size of the largest possible theoretical object allowed on a system (exception: it cannot be used to store addresses to member-function pointers). On a 32-bit system this will be a 32-bit value, on 64-bit it'll be a 64-bit value (as the type needs to be able to cover the whole memory range).